src/generators/static/packageFileGenerator.ts

```diff
@@ -49,52 +49,87 @@ export function generatePackageJson(
+        {
+          "path": "swagger/README.md",
+          "prefix": "package-version"
+        }
+      ]
+    },
+    engines: {
+      node: ">=12.0.0"
+    },
     scripts: {
-      test: 'echo "Error: no test specified" && exit 1',
-      build: "npm run clean && tsc --build && npm run extract-api",
-      "extract-api": "mkdirp ./review && api-extractor run --local",
-      pack: "npm pack 2>&1",
-      lint: "echo skipped",
-      audit: "echo skipped",
-      clean: "echo skipped",
-      "build:node": "echo skipped",
-      "build:browser": "echo skipped",
-      "build:test": "echo skipped",
+      audit: "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+      "build:browser": "echo skipped.",
+      "build:node": "echo skipped.",
       "build:samples": "echo skipped.",
-      "check-format": "echo skipped",
+      "build:test": "echo skipped.",
+      "build:debug": "echo skipped.",
+      "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+      clean: "rimraf dist dist-browser dist-esm test-dist temp types *.tgz *.log",
       "execute:samples": "echo skipped",
-      format: "echo skipped",
-      "test:node": "echo skipped",
+      "extract-api": "rimraf review && mkdirp ./review && api-extractor run --local",
+      "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+      "generate:client": "autorest --typescript swagger/README.md && npm run format",
+      "integration-test:browser": "echo skipped",
+      "integration-test:node": "echo skipped",
+      "integration-test": "echo skipped",
+      "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+      lint: "eslint package.json api-extractor.json src test --ext .ts",
+      pack: "npm pack 2>&1",
       "test:browser": "echo skipped",
+      "test:node": "echo skipped",
+      test: 'echo "Error: no test specified" && exit 1',
       "unit-test": "echo skipped",
       "unit-test:node": "echo skipped",
       "unit-test:browser": "echo skipped",
-      "integration-test:browser": "echo skipped",
-      "integration-test:node": "echo skipped",
-      "integration-test": "echo skipped",
-      docs: "echo skipped"
-    },
-    keywords: [],
-    author: "",
-    license: "MIT",
-    engines: {
-      node: ">=12.0.0"
+      docs: "typedoc --excludePrivate --excludeExternals  --out ./dist/docs ./src"
     },
+    "sideEffects": false,
+    "autoPublish": false,
     dependencies: {
-      "@azure-rest/core-client": "1.0.0-beta.8",
       "@azure/core-auth": "^1.3.0",
+      "@azure-rest/core-client": "1.0.0-beta.8",
       "@azure/core-rest-pipeline": "^1.3.0",
```

Comment:
What's the plan to keep these versions up to date with what the `azure-sdk-for-js` repo has? If we were generating these from tooling in the repo it would be easy to keep up to date

Reply:
it is smiliar to a placeholder here, and we will still update its version in automation tool